### PR TITLE
Added changes needed for setup with Flex

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://issues.ibexa.co/browse/EZP-32201

This PR contains various changes, but I tried grouping them by commits and reviewing them separately should be possible.

Things done:
1) https://github.com/ezsystems/BehatBundle/pull/174/commits/ce385ae6a9ac372093d457968d9942046c137609 - made GitHub token optional for fetching Pull Request data. This fixes unit tests - they were failing because the GitHub token used expired. This fix will be backported to lower branches.
2) https://github.com/ezsystems/BehatBundle/pull/174/commits/4a36b02cb6218a746cd03e697318d134d869e9c1 - moved the RSA key from metarepository (https://github.com/ezsystems/ezplatform/blob/master/bin/.travis/rsa_allure) to keep our Allure Reports working
3) https://github.com/ezsystems/BehatBundle/pull/174/commits/14f378d2ddf3aac3ddc57ded37658516a785cfaf - added a fallback for gathering application logs, the logfile name has changed - this config is no longer present: https://github.com/ezsystems/ezplatform/blob/master/config/packages/behat/monolog.yaml#L1-L6 so the filename is `behat.log` (used to be `travis_test.log`)
4) https://github.com/ezsystems/BehatBundle/pull/174/commits/bf16ad17c4dab9d21cacedddf492da5af1b7ad4b - switched Travis to use Symfony Flex during setup
5) https://github.com/ezsystems/BehatBundle/pull/174/commits/b200780c3d00d44b0935dfaa1b96a12be3c003e6 - Added some missing parameters from metarepository:
- https://github.com/ezsystems/ezplatform/blob/master/config/packages/behat/ezplatform.yaml#L6
- https://github.com/ezsystems/ezplatform-ee/blob/master/config/packages/behat/ezplatform.yaml#L23-L25
 This is a bit tricky, as the timeout parameter is already prepended in AdminUI and the only way to override is by using a Compiler pass. We could do it in a recipe, but I don't want to do that now (I'd like to avoid using recipes until they are really required and we get more practice with how they work).
I've added a new parameter that allows someone to disable this override if needed.

### CODE REVIEW

**Comment by bogusez** [comment]
@mnocon in this PR we also need additional spaces. It occurs in several lines for example 69.

File: .travis.yml [hunk 1]
```diff
@@ -62,22 +61,18 @@ before_install:
...
-  - if [ "${CHECK_CS}" == "1" -o "${PHPUNIT_CONFIG}" != "" ]; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
+  - travis_retry composer install --prefer-dist --no-interaction --no-suggest
   # Prepare whole environment if needed
-  - if [ "${BEHAT_OPTS}" != "" ]; then ./bin/.travis/prepare_ezplatform.sh ; fi
+  - if [ "${BEHAT_OPTS}" != "" ]; then ./vendor/bin/prepare_project_edition.sh oss ${PROJECT_VERSION} ${COMPOSE_FILE} ; fi
 
 before_script:
-  - if [ "${SETUP_BEHAT_OPTS}" != "" ]; then cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "bin/ezbehat $SETUP_BEHAT_OPTS" ; fi
-  - if [ "${SETUP_BEHAT_OPTS}" != "" ]; then cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "composer run post-install-cmd" ; fi
+  - if [ "${SETUP_BEHAT_OPTS}" != "" ]; then cd "$HOME/build/project"; docker-compose exec --user www-data app sh -c "vendor/bin/ezbehat $SETUP_BEHAT_OPTS" ; fi
```
Comment (bogusez) [inline .travis.yml:69]: space needed

**Review by bogusez — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mnocon) [reply .travis.yml]: Fixed in https://github.com/ezsystems/BehatBundle/pull/174/commits/586390935621b742d0135c30dc5b99bc49246186

**Review by bogusez — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Thanks! 💪

**Pull request merged**